### PR TITLE
Spanish

### PR DESCRIPTION
**Pull request by ploperav**
hi, i've added an es.ts file to provide a spanish translation. i'm from colombia, so if you're separating by region you should know that. thanks so much for doing this.

### CODE REVIEW

**Comment by ayjayt** [comment]
Thanks!

So I should mention (I didn't check otherwise) Paul is a translator not a programmer. So if there are strings that are not yet i18n'ed , they aren't added here. But if that's the case let's figure that out before we merge.

**Comment by ayjayt** [comment]
Hi @ploperav, I noticed a couple strings missing translations, just like 4 of them.
```
monitoring:
        "Monitoring {switch, select, true {on} false {off} other {unknown}}",

//need to translate "monitoring" "on" "off" "unknown"

"options-speed-label": "Play speed (aka, press 'Q' and 'E')",

"options-point-labels-before": valores anteriores (eg: "California, 163,696 square miles, 39 million people" ),
"options-point-labels-after": valores siguientes (eg: "163,696 square miles, 39 million people, California" )
```

File: src/translations/es.ts [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+    "options-hertz-lower": "Hertz más bajo",
+    "options-hertz-upper": "Hertz más alto",
+    "options-speed-label": "Play speed (aka, press 'Q' and 'E')",
+    "options-set-global":
+        "Guardar mis opciones para otros gráficos en esta página",
+    "options-use-continuous": "Usar modo continuo",
+    "options-continuous-descr":
+        "El modo continuo cambia cómo se reproducen los valores al presionar Shift+Home y Shift+End",
+    "options-point-labels": "Mostrar etiquetas del punto",
+    "options-point-labels-before": `valores anteriores (eg: "California, 163,696 square miles, 39 million people" )`,
```
Comment (julianna-langston) [inline src/translations/es.ts:178]: Do you mind translating the "square miles" and "million people", as well? Feel free to convert to km2, if you think that makes more sense.

File: src/translations/es.ts [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+    "key-monitor-toggle": "Cambiar modo de pantalla",
+    "key-dialog-help": "Abrir diálogo de ayuda",
+    "key-dialog-options": "Abrir diálogo de opciones",
+
+    "options-title": "Opciones",
+    "options-frontmatter":
+        "Al navegar este gráfico algunos sonidos pueden reusltar muy altos o muy bajos para que los escuche. Adicionalmente, si desea expandir el rango de sonidos disponibles, deslice los controles para ajustar el rango de sonido:",
+    "options-hertz-lower": "Hertz más bajo",
+    "options-hertz-upper": "Hertz más alto",
+    "options-speed-label": "Play speed (aka, press 'Q' and 'E')",
```
Comment (julianna-langston) [inline src/translations/es.ts:171]: Was this line intentionally left in english?

File: src/translations/es.ts [hunk 3]
```diff
@@ -0,0 +1,182 @@
+import type { translationDict } from ".";
+
+const dictionary: translationDict = {
+    description: "{title}, Gráfico Sonificado",
+    updated: "{title} Actualizado",
+    "updated-untitled": "Gráfico Actualizado",
+    missing: "Falta", // Faltante is better depending on context, where do we use this?
```
Comment (julianna-langston) [inline src/translations/es.ts:7]: This relates to missing data. The data points could be out of range, or perhaps not provided. For example, you might have a bar chart with bars like "Saturday, $100" and "Sunday, missing".

**Comment by julianna-langston** [comment]
> So I should mention (I didn't check otherwise) PaulA is a translator not a programmer. So if there are strings that are not yet i18n'ed , they aren't added here. But if that's the case let's figure that out before we merge.

We have i18n'd all of the user-facing messages, but not the error messages for developers. If you're interested, we can try to i18n the developer error messages in a few weeks, and we can let you know when those are ready for translation.

**Comment by julianna-langston** [comment]
@ploperav Thank you for all your work!

I'm looking at documenting the i18n support, and I'd like to add a chart in Spanish. The chart will have:

- Title = "Moons per Planet"
- X axis = "Planets"
- Y axis = "Number of Moons"
and then the planets.

Do you mind translating those phrases for me?

(I assume that I can use these names for the planets: Mercurio, Venus, Tierra, Marte, Júpiter, Saturno, Urano, Neptuno)

**Comment by julianna-langston** [comment]
We have published Chart2Music, v1.14.0, which includes this feature. Your changes are now live.

Thanks again for all your help!

**Pull request merged**